examples/tensorflow_distributed/tensorflow_distributed.py

```diff
@@ -0,0 +1,78 @@
+# A basic example showing how to use Runhouse to run a TensorFlow distributed training script on a cluster of GPUs
+
+import asyncio
+import json
+import os
+
+import runhouse as rh
+import tensorflow as tf
+
+
+# Define the Tensorflow distributed training script
+def train():
+    # Initialize the distributed training environment,
+    # per https://www.tensorflow.org/tutorials/distribute/multi_worker_with_keras
+    tf_config = json.loads(os.environ["TF_CONFIG"])
+    strategy = tf.distribute.MultiWorkerMirroredStrategy()
+    num_workers = strategy.num_replicas_in_sync
+    print(f"Worker {tf_config['task']['index']} of {num_workers} initialized")
+
+    # Create a simple model and optimizer
+    model = tf.keras.Sequential([tf.keras.layers.Dense(10, activation="relu")])
+    optimizer = tf.keras.optimizers.SGD(0.01)
+
+    with strategy.scope():
+        model.compile(optimizer=optimizer, loss="mse")
+
+    model.fit(
+        tf.data.Dataset.from_tensor_slices(
+            (tf.random.normal([1000, 10]), tf.random.normal([1000, 1]))
+        ).batch(32)
+    )
+
+    print(f"Worker {tf_config['task']['index']} finished")
+
+
```

Comment:
Maybe helpful to add the usual callout:
```
# :::note{.info title="Note"}
# Make sure that your code runs within a `if __name__ == "__main__":` block, as shown below. Otherwise,
# the script code will run when Runhouse attempts to run code remotely.
# :::
```